Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM node:6 AS builder
+
+RUN groupadd --gid 1100 lisk && \
+    useradd --create-home --home-dir /home/lisk --shell /bin/bash --uid 1100 --gid 1100 lisk
+COPY . /home/lisk/lisk/
+RUN chown lisk:lisk --recursive /home/lisk/lisk
+
+USER lisk
+WORKDIR /home/lisk/lisk
+
+RUN npm install
+RUN ./node_modules/.bin/grunt release
```

Comment:
What's the purpose of doing grunt release. I see that it adds files that mark the version which can help identify the version but it also creates a packaged tarball and copies it to a release directory creating unnecessary bulk for the docker image.

If the build and revision files are necessary grunt build/revision should be enough and not create unnecessary files.

Reply:
@diego-G can you chime in on this?

Reply:
@fchavant I agree we don't need grunt release in the docker image. Regarding the other comment, I prefer to keep `npm install` without `--production` flag. But It is true we should use `NODE_ENV` environment variable to signalise what kind of installation we want in the image. By default `npm` listens it, not installing devDependencies if it is set to `production`.